common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -53,7 +54,8 @@ case class ActivationMessage(
     activationId: ActivationId,
     activationNamespace: EntityPath,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    traceMetadata: Option[SpanMetadata] = None)
```

Comment:
> The point is to reduce the control plane to just control. 

I forgot to give this a big 👍 . Can you share the issue # ? I'd be interested to understand the thinking and separation between what's _data plane_ vs _control plane_.